concepts/80 Troubleshooting/00 Basics/10 Check Requests in the Network Tab.md

```diff
@@ -0,0 +1,23 @@
+DevExtreme data-bound components may not display data or display unexpected data. The components use network requests to communicate with servers.
```

Comment:
The paragraph seems a bit incomplete to me. I would add a sentence in the end, just to finalize the thought. 

```suggestion
DevExtreme data-bound components may not display data or display unexpected data. The components use network requests to communicate with servers. You may need to resolve network communication issues.
```